gradle/libs.versions.toml

```diff
@@ -57,9 +57,10 @@ jackson213 = { strictly = "2.13.4"}
 jackson214 = { strictly = "2.14.2"}
 jackson215 = { strictly = "2.15.2"}
 jakarta-el-api = "3.0.3"
+jakarta-servlet-api = "6.1.0"
 jaxb-api = "2.3.1"
 jaxb-runtime = "2.3.9"
-jetty = "9.4.55.v20240627"
+jetty = "11.0.22"
```

Comment:
Worth adding code comment like "this is the latest Jetty version that runs with JDK11"  here?

Reply:
we have a bunch of dependencies like this where we're at the latest version for the JDK we're running with, so I don't think it's worth adding such a comment just here.
That being said, we might actually need to re-visit and see if we're still at the latest version of each dependency for JDK11 now (I'll open a separate issue to tackle that, since dependabot doesn't take the JDK version into account)

Reply:
https://github.com/apache/iceberg/issues/10852